.golangci.yml

```diff
@@ -52,5 +52,5 @@ issues:
   max-same-issues: 0
 
 service:
-  golangci-lint-version: 1.27.0 # use the fixed version to not introduce new linters unexpectedly
+  golangci-lint-version: 1.44.2 # use the fixed version to not introduce new linters unexpectedly
```

Comment:
As we have verified `"-Dzstd_SOURCE=BUNDLED"` is the key to fix the code checker on mac, so please revert this back ,thanks. https://github.com/milvus-io/milvus/runs/5531761338?check_suite_focus=true